cucumber-selenium-tests/src/test/java/com/skillstorm/Utilities/Navigator.java

```diff
@@ -36,7 +36,7 @@ public class Navigator {
     public static final String URL_TAX= "";
     public static final String URL_TAXEDITVIEW= "";
     public static final String URL_TAXRESULTS= "";
-    public static final String URL_ERROR= "";
+    public static final String URL_ERROR= "http://localhost:5173/error";
```

Comment:
For URL_ERROR, I'd suggest pulling latest and then using the convention of:
URL + "/error";